From f12f33d326c0ee3a3f75d74a672417bf4f418784 Mon Sep 17 00:00:00 2001
From: Kevin Jones <kevin@vcsjones.com>
Date: Fri, 12 Jun 2026 14:06:50 -0400
Subject: [PATCH 7/9] Fix if that should have been an ifdef on non-portable
 OpenSSL 1.1.1 builds

(cherry picked from commit b9ff865efa9e59e54eaac6dab9ef6ea071fad2c1)
---
 .../pal_ecc_import_export.c                               | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/native/libs/System.Security.Cryptography.Native/pal_ecc_import_export.c b/src/native/libs/System.Security.Cryptography.Native/pal_ecc_import_export.c
index 268d6f3e7f4..5b862e8e692 100644
--- a/src/native/libs/System.Security.Cryptography.Native/pal_ecc_import_export.c
+++ b/src/native/libs/System.Security.Cryptography.Native/pal_ecc_import_export.c
@@ -560,7 +560,7 @@ int32_t CryptoNative_EvpPKeyEcHasExplicitEncoding(EVP_PKEY* pkey)
         char encoding[32] = {0};
         if (!EVP_PKEY_get_utf8_string_param(pkey, OSSL_PKEY_PARAM_EC_ENCODING, encoding, sizeof(encoding), NULL))
             return 0;
-        
+
         return (strcmp(encoding, OSSL_PKEY_EC_ENCODING_EXPLICIT) == 0) ? 1 : 0;
     }
 #endif
@@ -1511,7 +1511,7 @@ int32_t CryptoNative_EvpPKeyGenerateByEcCurveOid(
 
 #ifdef NEED_OPENSSL_1_1
     // The portable build should only use the legacy OSSL 1.1 code path if OSSL 3.0+ APIs aren't present.
-#if FEATURE_DISTRO_AGNOSTIC_SSL
+#ifdef FEATURE_DISTRO_AGNOSTIC_SSL
     if (rc == 3)
 #endif
     {
@@ -1718,7 +1718,7 @@ int32_t CryptoNative_EvpPKeyCreateByEcParameters(
 
 #ifdef NEED_OPENSSL_1_1
     // The portable build should only use the legacy OSSL 1.1 code path if OSSL 3.0+ APIs aren't present.
-#if FEATURE_DISTRO_AGNOSTIC_SSL
+#ifdef FEATURE_DISTRO_AGNOSTIC_SSL
     if (rc == 3)
 #endif
     {
@@ -2085,7 +2085,7 @@ int32_t CryptoNative_EvpPKeyCreateByEcExplicitParameters(
 
 #ifdef NEED_OPENSSL_1_1
     // The portable build should only use the legacy OSSL 1.1 code path if OSSL 3.0+ APIs aren't present.
-#if FEATURE_DISTRO_AGNOSTIC_SSL
+#ifdef FEATURE_DISTRO_AGNOSTIC_SSL
     if (rc == 3)
 #endif
     {
-- 
2.53.0

